setup.py

```diff
@@ -29,7 +29,7 @@
 
 # Extra third-party libraries
 extras_require['extras'] = extras_require['examples']+[
-    'cyordereddict', 'flexx==0.4.1']
+    'cyordereddict', 'flexx==0.8.0']
```

Comment:
I don't think flexx is required for anything else so this could be dropped entirely and replaced with pscript.